mobile/src/components/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pressable, Text, View } from "react-native";
+/**
+ * name - name of the button.
+ * addButtonStyle - add custom style to the button.
+ * addTextStyle - add custom style to the text.
+ * pilled - make the button pilled.
+ **/
+export default function Buttons({
+  name,
+  addButtonStyle,
+  addTextStyle,
```

Comment:
I'm not a huge fan of this API. People trying to use your button have to learn about these special style props. I think it would be better if they could just use regular `className`. I would remove the ability to style the `Text` and just provide `className` that you add to the `Pressable`. You could use the `children` prop so they could pass in their own `Text` if they want to style that.

Something like:
```ts
export default function Button({
	name, className, children
}) {
	// ...
	return (
		<Pressable className=`${baseStyle} ${className}`>
			{name && <Text className={textStyle}></Text>}
			{children}
		</Pressable>
	)
}
```